cleanlab/rank.py

```diff
@@ -23,7 +23,7 @@
 every example. You can then do something like: `np.argsort(label_quality_score)` to obtain ranked
 indices of individual data.
 
-If you aren't sure which method to use, try `get_normalized_margin_for_each_label()`.
+If you aren't sure which method to use, try `get_self_confidence_for_each_label()`.
```

Comment:
Let's delete this sentence. The novice user is supposed to use `get_label_quality_scores` with the provided default in there.

Reply:
Agree, sounds good. Deleted the sentence.

c936311ee36ff263796a4bdc3a982e84ed189605